.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: node_js
 node_js:
-- 0.10
+  - 0.10
+before_install:
+  - gem install sass
```

Comment:
Right now this PR is using grunt-contrib-sass, which uses [Ruby sass](https://github.com/gruntjs/grunt-contrib-sass/issues/38). However, we can try using [grunt-sass](https://github.com/sindresorhus/grunt-sass) instead.